Ada Examples/Examples Based On OpenGL Superbible/Simplest/src/render.ads

```diff
@@ -0,0 +1,11 @@
+with System;
+
+with GL.Objects.Programs;
+
+with Glfw;
+
+package Render is
```

Comment:
Why is this a package when it contains only one procedure? `Render.Render` is not a good name an can be avoided. Just omit the package and the `.ads` and directly have `procedure Render (...) is ... end Render` at top level.